Conference: Call for abstracts and intoduction text (#432)
nordic-rse/nordic-rse.github.io: content/events/2024-in-person-conference/_index.md

# Nordic-RSE in person conference 2024 <span style="color: gray;">#NordicRSEconf</span>

**May 30-31**

Registration will open in January 2024

## What is it?

This is the first Nordic RSE in person conference. We are an association of researchers who use software and people who write
that software. So this is not like most academic conferences in that the focus is not on the results of research, but rather
on the tools that underlie those results.

Most research is already built on a foundation of sofware tools and the quality and reliability of those tools is key to
producing reliable results. And the importance of software keeps growing. In this conference we want to talk about what software
you are use for your research and how you use it, the software you develop and how you go about developing it.

## Where

> At Aalto University, Espoo, Finland.

Reviewer: ```suggestion
On the campus of Aalto University in Otaniemi, Espoo, Finland.
```

### How you can help

>  - Submit a contribution! The conference will be more interesting with your contribution on the agenda.
>  - [Share the announcement with others](/events/2023-online-unconference/share/). 

Reviewer: ```suggestion
 - Submit a contribution! The conference will be more interesting with your contribution on the agenda.
 - Encourage your colleagues to submit a contribution. Since the term RSE is still fairly new, they might not know about it yet and be hesitant to submit something. 
 - [Share the announcement with others](/events/2023-online-unconference/share/). 
```

## Call for Contributions

> The call for contributions is open at URL_HERE.

Reviewer: ```suggestion
The call for contributions is open at URL_HERE.
```
TODO?

> The call for contributions is open at URL_HERE.

Reviewer: TODO?

Author: Yes, we need the Indigo form before this goes public :)

> We invite you to submit contributions on any topics related to software. Whether you are using it for research or developing it yourself,

Reviewer: ```suggestion
We invite you to submit contributions on any topics related to research software. Whether you are using it or developing it yourself,
```

> we are interested in your experience and expertice. Topics are divided roughly into 3 sessions, "Best Practices", "Techical" and

Reviewer: Should we also invite people to talk about research support, mentoring and training? 
It could be very interesting to also have these topics represented :nerd_face:

> "Soft Skills". Don't let this limit you, any topics are welcome.

Reviewer: ```suggestion
"Soft Skills". But don't let this limit you, many topics are welcome. Just submit your idea and we can discuss and find a spot for it!
```

# Nordic-RSE in person conference 2024 <span style="color: gray;">#NordicRSEconf</span>

**May 30-31**

Registration will open in January 2024

## What is it?

This is the first Nordic RSE in person conference. We are an association of researchers who use software and people who write
that software. So this is not like most academic conferences in that the focus is not on the results of research, but rather
on the tools that underlie those results.

Most research is already built on a foundation of sofware tools and the quality and reliability of those tools is key to

> producing reliable results. And the importance of software keeps growing. In this conference we want to talk about what software

Reviewer 2: I would say "In this conference we want to talk about what pipelines you develop and use for your research, and how you use it, the software you develop, and how you got about developing it".

The reason is that for some of us it might be not only software, but also hardware that they develop.

you use for your research and how you use it, the software you develop and how you go about developing it.

## Where

> On the campus of Aalto University in Otaniemi, Espoo, Finland.

Reviewer 2: Possibly add hyper link to maps?

### How you can help

 - Submit a contribution! The conference will be more interesting with your contribution on the agenda.
 - Encourage your colleagues to submit a contribution. Since the term RSE is still fairly new, they might not know about it yet and be hesitant to submit something. 
 - [Share the announcement with others](/events/2024-in-person-conference/). 

> 

Reviewer 2: I would either repeat or just write here the topic of the contributions (pipelines we use). This for people who skip the introduction.

Reviewer 2: Oh sorry I notice that this comes below. Maybe a hyper link to that part of the document along with "submit"?

"Soft Skills". But don't let this limit you, many topics are welcome. Just submit your idea and we can discuss and find a spot for it!

All formats of communication are welcome (or check what we have done [in the unconference](/events/2023-online-unconference/)):
- *Discussion topics*: is software, research software community or research software engineering something close to your heart? Come chat with other enthusiasts!
- *Demonstrations:* show us some tools or software that you like
- *Workshops or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (max 3 hours in a day)
- *Talks*: teach us something new
- *On-going projects*: share something unfinished or almost working that you would like to get feedback on
- *Something else*: is there something you would like to share that
  does not fit any of the previous points? We are looking forward to hearing
  about it!

The open session will include on day contributions, topics that arise during the conference.

And of course we will all follow a [Code of Conduct](https://nordic-rse.org/about/code-of-conduct/).

### Price

> The participation fee will be 50€-100€ and will cover a conference dinner on Thursday.

Reviewer 2: "and will _also_ cover"

Otherwise it sounds like that's the only thing it will be covered, which for 50-100€ it's a lot. Better yet if we name two or three things that will be covered.

Author: Currently reads "will also cover lunch and refreshments". Could add venue, but that seems obvious.

Author: Updated following the last comment and ended up removing this part.

## Program

### Day 1: 2024-5-30

All times are EET ([convert to local time](https://arewemeetingyet.com/Helsinki/2024-5-30/9:00))

> 

Reviewer 2: Maybe add "This schedule will be subjected to review after the call of abstracts is closed". or similar

- 9:00 - 11:00 : Informal get-together on campus.

> - 11:00 - 12:00 : Lunch

Reviewer 2: Is there a reason why Lunch is between 11 and 12, and not 12 and 13 as it would be expected? Or maybe Finnish food times are different?

Author: Partly Finnish lunch time, partly because the queues are much longer at 12 if we go to a restaurant.

Author: This can be changed if we decide to order catering.

We invite you to submit contributions on any topics related to research software. Whether you are using it or developing it yourself,
we are interested in your experience and expertise. Topics are divided roughly into 3 sessions, "Best Practices", "Technical" and
"Soft Skills". But don't let this limit you, many topics are welcome. Just submit your idea and we can discuss and find a spot for it!

All formats of communication are welcome (or check what we have done [in the unconference](/events/2023-online-unconference/)):
- *Discussion topics*: is software, research software community or research software engineering something close to your heart? Come chat with other enthusiasts!
- *Demonstrations:* show us some tools or software that you like
- *Workshops or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (max 3 hours in a day)
- *Talks*: teach us something new
- *On-going projects*: share something unfinished or almost working that you would like to get feedback on
- *Something else*: is there something you would like to share that
  does not fit any of the previous points? We are looking forward to hearing
  about it!

> The open session will include on day contributions, topics that arise during the conference.

Reviewer: ```suggestion
The open (or 'unconference') session will include on day contributions, topics that arise during the conference.
```
Some may still know the term but not understand the open as such 🤔

And of course we will all follow a [Code of Conduct](https://nordic-rse.org/about/code-of-conduct/).

### Price

Because this is the first in person Nordic RSE conference, the participation fee still needs to be estimated based on the interest that we see in the community. The fee will be around 50€ - with options for discounts for individual cases.

## Program

> The schedule subject to change untill the submitted contributions are reviewed.

Reviewer: ```suggestion
The schedule is subject to change until the submitted contributions are reviewed.
```

### Day 1: 2024-5-30

All times are EET ([convert to local time](https://arewemeetingyet.com/Helsinki/2024-5-30/9:00))

- 9:00 - 11:00 : Informal get-together on campus.
- 11:00 - 12:00 : Lunch
- 12:00 - 13:20 : Best practices session
- 13:20 - 13:40 : Break
- 13:40 - 16:00 : Soft skills session

### Day 2: 2024-5-31

All times are EET ([convert to local time](https://arewemeetingyet.com/Helsinki/2024-5-30/9:00))

- 9:00 - 11:00 : Technical session 
- 11:00 - 12:00 : Lunch
- 12:00 - 13:00 : Technical session continued

> - 13:00 - 16:00 : Open session

Reviewer: ```suggestion
- 13:00 - 13.20 : Break
- 13.20 - 16:00 : Open session
```
Probably wrong time, but since the break is in the schedule on day 1 also

# Nordic-RSE in person conference 2024 <span style="color: gray;">#NordicRSEconf</span>

**May 30-31**

Registration will open in January 2024

## What is it?

This is the first Nordic RSE in person conference. We are an association of researchers who use software and people who write
that software. So this is not like most academic conferences in that the focus is not on the results of research, but rather
on the tools that underlie those results.

Most research is already built on a foundation of sofware tools and the quality and reliability of those tools is key to
producing reliable results. And the importance of software keeps growing. In this conference we want to talk about what
pipelines you develop and use for your research, and how you use it, the software you develop, and how you got about
developing it

## Where

On the [campus of Aalto University](https://maps.app.goo.gl/3TGvPnBw254T5X817) in Otaniemi, Espoo, Finland.

### How you can help

>  - [Submit a contribution!](http://127.0.0.1:1111/events/2024-in-person-conference#call-for-contributions) The conference will be more interesting with your contribution on the agenda.

Reviewer 2: Should this just '/events/2024-in-person-conference#call-for-contributions')?

Author: Whoops, thanks!
